core/parameters/res/runtime_configs/parameters.yaml

```diff
@@ -12,6 +12,9 @@ pessimistic_gas_price_inflation: {
   denominator: 100,
 }
 
+# Stateless validation config
+state_witness_size_soft_limit: 1_000_000_000_000
```

Comment:
why are we using the old(?) value here?

Reply:
My understanding is that the runtime config is the one controlling the behavior; if we use the new value here it would break the existing runtime.

Reply:
Correct, this is the base parameter, applied to all versions starting at genesis up to the protocol version at which the parameter is overridden. I would given that the parameter is an `usize`, I would perhaps look at making it `0xFFFF_FFFF_FFFF_FFFF`, much like its done for `RuntimeFeesConfig::test`, so that there's no way in the world this limit is reached. Admittedly 10¹² is already huge enough to cover our bases, but the round number kinda makes one think that this is a proper parameter value, which can be slightly confusing to those who don't know the background behind this parameter.

Reply:
for some reason parsing 0xFFFF_FFFF_FFFF_FFFF did not work so I used 999_999_999_999_999 instead.